src/commands/add.ts

```diff
@@ -60,43 +59,53 @@ function askPrompt(questions: Questions): TaskEither<unknown, Answers> {
 }
 
 function askWhere(): TaskEither<unknown, string> {
-  return askPrompt([
-    {
-      type: "input",
-      name: "where",
-      message: "Where are you going?"
-    }
-  ]).map(a => a.where);
+  return taskEither.map(
```

Comment:
😕again, reversed business logic, where asking an input and __then__ mapping, but we're forced to write it in reverse order